shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/main/java/org/apache/shardingsphere/data/pipeline/api/job/progress/JobProgress.java

```diff
@@ -46,19 +41,17 @@ public final class JobProgress implements PipelineJobProgress {
     
     private boolean active;
     
-    private Map<String, InventoryTaskProgress> inventoryTaskProgressMap;
+    private InventoryTaskProgress inventory;
     
-    private Map<String, IncrementalTaskProgress> incrementalTaskProgressMap;
+    private IncrementalTaskProgress incremental;
```

Comment:
After reading through the changes, I changed some ideas. In order to keep it more understandable, we'd better not change the meaning of TaskProgress implementations and not add XxxTaskProgressItem.

Possible changes:

1, Could we keep InventoryTaskProgress and IncrementalTaskProgress unchanged? Since InventoryTask.getProgress() return InventoryTaskProgress and IncrementalTask.getProgress() return IncrementalTaskProgress.

2, We just add new classes, e.g.
- InventoryTaskProgresses to encapsulate `Map<String, InventoryTaskProgress> inventoryTaskProgressMap`
- IncrementalTaskProgresses to encapsulate `Map<String, IncrementalTaskProgress> incrementalTaskProgressMap`

Reply:
OK, i try to fix it

Reply:
I will add new class `JobInventoryTaskProgress` like this, `JobIncrementalTaskProgress` same as。 
```java
public class JobInventoryTaskProgress {

    private final Map<String, InventoryTaskProgress> inventoryTaskProgressMap;
}
```
Is the direction correct？

Reply:
Yes, that's OK.